spec/rails_helper.rb

```diff
@@ -44,3 +45,7 @@
     with.library :rails
   end
 end
+
+Fictium.configure do |config|
+  config.fixture_path = File.join(__dir__, 'support', 'docs')
```

Comment:
I would add to the bootstrap:

```rb
config.exporters << Fictium::Postman::V2Exporter.new
```

As we will almost 100% of the time would like to export to Postman on our projects.